README.md

```diff
@@ -1,14 +1,17 @@
-# drg-mod-integration
-3rd party mod integration tool to download and integrate mods completely
-externally of the game. This enables more stable mod usage as well as offline
-mod usage. Works for both Steam and Microsoft Store versions.
+# Deep Rock Galactic Mod Integration
```

Comment:
I would prefer the title remains the crate name and not have the full game name in the title. Even 'drg' is more than I'd like but naming things is hard...